src/main/scala/com/databricks/spark/avro/package.scala

```diff
@@ -31,6 +31,9 @@ package object avro {
    * Adds a method, `saveAsAvroFile`, to DataFrame that allows you to save it as avro file.
    */
   implicit class AvroDataFrame(dataFrame: DataFrame) {
-    def saveAsAvroFile(path: String): Unit = AvroSaver.save(dataFrame, path)
+    def saveAsAvroFile(
+       path: String,
+       recordName: String = "topLevelRecord",
+       recordNamespace: String = ""): Unit = AvroSaver.save(dataFrame, path)
```

Comment:
If `save()` changes, this will have to be changed to take `Map[String, String]` as well. Also, can you please update the README to mention the updated API.